src/panel/layouts.js

```diff
@@ -54,8 +61,9 @@ export function getVisibleBbox(mb) {
   return bbox;
 }
 
-export function getMapCenterOffset({ isMobile }) {
-  return isMobile ? [0, 0] : [(DESKTOP_PANEL_WIDTH + ADDITIONAL_PADDING) / 2, 0];
+// Disable offset on mobile and in iframe mode
```

Comment:
It's strange to have this comment here whereas the code deciding that is elsewhere, in `scene.js`. It would be consistent with the other functions to receive `isIframe` here alongside `isMobile` as param.

Reply:
Yes, I did that then thought it was a bit absurd to test both isMobile && isIframe for the same outcome ( [0,0] ).
But ok!